config/locales/qqq.yml

```diff
@@ -406,7 +406,9 @@ qqq:
     join_prompt: |-
       Do you want to join this course?
       {{Identical|Join}}
-    join_successful: Message that join was successful
+    join_successful: Message that join was successful and explains the functioning of Dashboard
+      with emphasis on how Editathons are updated every 5 minutes while other course level
```

Comment:
This is not accurate after the changes above.

Reply:
What would you have it changed to instead? Something more precise?

Reply:
Something more vague, I think. "Message that joining a program was successful, with an explanation of how the Dashboard will track contributions."